### PR TITLE
Hotfix/validation

### PR DESCRIPTION
**Pull request by shyambhongle**
Fixes #

Changes in this pull request:
Validation fixes
Intervention preview fix

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "expo": {
     "name": "TreeMapper",
     "slug": "treemapper",
-    "version": "2.0.1",
+    "version": "2.0.3",
```
Comment (norbertschuler) [inline app.json:5]: Don't forget to update this number also in the iOS project and Android build file for the native apps.

**Review by norbertschuler — APPROVED** [review]
I tested the possible entry of "," instead of "." and it seems to work well for single or multiple trees as well as for editing the values afterwards. I have not tested the other changes.
(btw. I still cannot build the iOS app for the same error as before `error: unsupported option '-G' for target 'arm64-apple-ios13.4' (in target 'RNZipArchive' from project 'Pods')`)

File: app.json (hunk 1 above)
Comment (norbertschuler) [reply app.json:5]: If expo is generating these file on the fly and using this info for it, it probably isn't necessary any more to care about this manually.

Comment (shyambhongle) [reply app.json:5]: Yes

**Comment by shyambhongle** [comment]
Hi,
The build issue is fixed in the latest version of the zip package, but it requires setting the minimum deployment version to 15.5. Since we are using Expo and the iOS/Android folders are created dynamically, I will need to write a plugin to handle this. 
I am currently making the necessary changes in this [branch](https://github.com/Plant-for-the-Planet-org/treemapper/tree/bugfix/xcode-build-fix). I think It's better to merge the branch as writing plugin can take some time.

**Pull request merged**